src/coq/decompiler/decompiler.ml

```diff
@@ -187,13 +233,22 @@ let dot tac next = Some (Compose ([ tac ], [ next ]))
 let qed tac = Some (Compose ([ tac ], []))
 
 (* Inserts "simpl." before every rewrite. *)
-let rec simpl (t : tactical) : tactical =
+let rec simpl sigma (t : tactical) : tactical =
```

Comment:
As a rule of thumb, when you update an `evar_map`, you usually want to return it. I think that applies here.